libri-whitepaper.tex

```diff
@@ -73,23 +75,24 @@ \subsection{Decentralized storage}
 
-IPFS \cite{ipfs} uses much of the same design as BitTorrent---peer-to-peer file sharing, possible caching of popular documents, tit-for-tat incentive accounting---but it addresses the data stored by a hash of its content instead of a filename. This content-addressing combined with a simple link structure allows it to behave as a Merkle DAG, giving it great flexibility in being able to store and model many forms of data, from simple blobs to files to whole filesystems. At its heart, though, IPFS looks very much like BitTorrent: peers host data that others may optionally copy and host as well, and the addresses of the peers hosting each object are stored in a Kademlia DHT.
+% Jamie edit (para below): Needed to transition from the data-loss problem to the incentive problem. I make an aggressive claim that you should review (churn and thus data loss corresponds with lack of incentives, therefore, we need incentives).
 
-Peer-to-peer data storage and sharing networks are powerful, but the networks themselves are unable to offer any guarantees about the data stored in them. If the only peer hosting some data goes offline, that data is lost to the network. Over the last few years, blockchain-based networks like Sia \cite{sia}, Storj \cite{storj}, \cite{bigchaindb,bigchaindb2}, Ethereum Swarm \cite{swarm1,swarm2}, and Filecoin \cite{filecoin} have been developed in an attempt to solve the incentives problem: why should any peer dedicate storage space, bandwidth, and machine resources? These different networks each tackle this problem in slightly different ways, but they all rely on an economic incentive for peers to store (and keep storing) data. This economic incentive requires transactions betwen parties, and those transactions require a blockchain. 
+Peer-to-peer data storage and sharing networks are powerful, but the networks themselves are unable to offer any guarantees about the data stored in them. If the only peer hosting a particular segment of data goes offline, that data is lost from the network. Absent sufficient incentives to operate a storage node, there is high churn and little reliability in the network. Over the last few years, blockchain-based networks such as Sia \cite{sia}, Storj \cite{storj}, \cite{bigchaindb,bigchaindb2}, Ethereum Swarm \cite{swarm1,swarm2}, and Filecoin \cite{filecoin} have been developed in an attempt to solve the incentives problem: why should any peer dedicate storage space, bandwidth, and machine resources? These different networks each tackle this problem in slightly different ways, but they all rely on an economic incentive for peers to store (and keep storing) data. This economic incentive requires monetary transactions between parties, and those transactions require a blockchain. 
```

Comment:
`there is high churn and little reliability in the network.` -> `networks are unable to offer durability and performance guarantees required for most production-level storage needs.`